lua/nvim-treesitter/configs.lua

```diff
@@ -279,13 +279,13 @@ function M.setup(user_data)
     else
       config.modules[name] = vim.tbl_deep_extend('force', config.modules[name] or {}, data)
 
-      recurse_modules(function(mod_name)
+      recurse_modules(function(mod_name, module, new_path)
```

Comment:
Since `module` is unused can you change it to `_`

Reply:
Done. I'm new to lua, but I think I can apply the same change to `mod_name` as well?